src/components/MessageBox.js

```diff
@@ -0,0 +1,91 @@
+const React = require('react');
+const _merge = require('lodash/merge');
+
+const StyleConstants = require('../constants/Style');
+const Icon = require('../components/Icon');
+
+const MessageBox = React.createClass({
+  propTypes: {
+    children: React.PropTypes.node,
+    color: React.PropTypes.string,
+    icon: React.PropTypes.string,
+    styles: React.PropTypes.object,
+    title: React.PropTypes.string
+  },
+
+  getInitialState () {
+    return {
+      isOpen: false
```

Comment:
@derek-boman, @lukeschunk: should the `MessageBox` default to the open state or closed? I think it should be open by default so I'm curious to hear your opinion on it.